frontend/src/components/Scroll/ChevronScroll.tsx

```diff
@@ -50,13 +50,10 @@ function ChevronScroll(props: React.PropsWithChildren<{}>) {
   };
 
   return (
-    <div className="chevron-scroll-outer-container">
+    <div className="whitespace-nowrap flex w-[1088px] h-[47px]">
       <button
         className={combineClasses(
-          "chevron-scroll-left-btn",
-          "align-center",
```

Comment:
We should convert this `align-center` class to tailwind. It can be found in the `_alignment.scss` file.

CSS:
align-items: center